core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -313,6 +320,17 @@ private[chisel3] object Converter {
               stmts = frame.outer
               stmts += when
               scope = scope.tail
+            case GroupDefBegin(info, declaration) =>
+              val groupDefine = fir.GroupDefine(convert(info), declaration.name, fir.EmptyStmt)
+              val frame = GroupFrame(groupDefine, stmts)
+              stmts = new VectorBuilder[fir.Statement]
+              scope = frame :: scope
+            case GroupDefEnd(info) =>
+              val frame = scope.head.asInstanceOf[GroupFrame]
+              val groupDefine = frame.group.copy(body = fir.Block(stmts.result()))
+              stmts = frame.outer
+              stmts += groupDefine
+              scope = scope.tail
           }
```

Comment:
I think there is a bug here where this is not properly terminating the group. I'm seeing the problem where anything after the final group are not showing up in the emitted CHIRRTL.

Reply:
The bug wasn't here, but in the Serializer. Fixed in https://github.com/chipsalliance/chisel/pull/3541/commits/8dfc4ed9e8874cdd9bbd38af43db65556be132e6.